docker-compose.yml

```diff
@@ -26,6 +26,8 @@ services:
     build:
       context: .
       dockerfile: Dockerfiles/Dockerfile.bcda
+    env_file:
+      - ./shared_files/config/local.env
```

Comment:
I like this approach, but I'm feeling like it obfuscates the configuration a little bit.  For example, if I were looking at the `docker-compose` file only, I would have to dig to determine where/how certain vars like `BB_HASH_ITER` and `BB_HASH_PEPPER` are being set.  Maybe we can have comments in the `docker-compose` indicating exactly which vars are being set from `local.env`?

Reply:
Great idea.  I've done this in the latest commit.